static/app/components/events/interfaces/spans/scrollbarManager.tsx

```diff
@@ -473,6 +468,49 @@ export class Provider extends Component<Props, State> {
     }
   };
 
+  markSpanOutOfView = (spanId: string) => {
+    if (!this.spansInView.removeSpan(spanId)) {
+      return;
+    }
+
+    const left = this.spansInView.getScrollVal();
```

Comment:
I think you might want to calc `left` inside the `performScroll` since it would be the most up to date there? It might not matter functionally though, it does save calling it twice.

Reply:
Hmmm, this makes sense, but `performScroll` would have to be restructured since it's also being used for the manual scrolling/wheeling. As you said functionally it wouldn't make a difference, and I think it would require some unnecessary effort to facilitate it